apps/dashboard/tests/dashboard-created-flow.spec.ts

```diff
@@ -0,0 +1,115 @@
+import { StepTypeEnum } from '@novu/shared';
```

Comment:
How about naming it from the user journey perspective? For example: `manage-workflows.e2e.ts`.

I added the e2e extension as we have the convention with e2e testing file naming on the API.